Binance.Net/Objects/Spot/MarginData/BinanceInterestHistoryResult.cs

```diff
@@ -26,6 +26,7 @@ public class BinanceInterestHistory
         /// <summary>
         /// Interest rate
         /// </summary>
+        [JsonProperty("interestRate")]
```

Comment:
Then it would seem there is a bug in it somewhere else in the deserializer because it isn't matching this, 

I had to change this to get it to work.

Am I missing something else?